<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserKeyIpFilterWidget</class>
 <widget class="QWidget" name="UserKeyIpFilterWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>567</width>
    <height>256</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>User informations</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="1">
    <widget class="QRadioButton" name="rbtn_whitelist">
     <property name="text">
      <string>Liste blanche</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QRadioButton" name="rbtn_no_filter">
     <property name="text">
      <string>Pas de filtrage</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QRadioButton" name="rbtn_blacklist">
     <property name="text">
      <string>Liste noire</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0" colspan="4">
    <widget class="QStackedWidget" name="stack_filter">
     <widget class="QWidget" name="page_no_filter"/>
     <widget class="QWidget" name="page_whitelist">
      <layout class="QGridLayout" name="gridLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item row="0" column="0" colspan="2">
        <widget class="QTableWidget" name="tbw_whitelist"/>
       </item>
       <item row="1" column="0">
        <widget class="QLineEdit" name="lne_whitelist"/>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="btn_add_whitelist">
         <property name="text">
          <string>Ajouter</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_blacklist">
      <layout class="QGridLayout" name="gridLayout_2">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item row="0" column="0" colspan="2">
        <widget class="QTableWidget" name="tbw_blacklist"/>
       </item>
       <item row="1" column="0">
        <widget class="QLineEdit" name="lne_blacklist"/>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="btn_add_blacklist">
         <property name="text">
          <string>Ajouter</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
